packages/x-components/src/x-modules/next-queries/components/next-queries.vue

```diff
@@ -100,16 +100,52 @@
 You don't need to pass any props, or slots. Simply add the component, and when it has any next
 queries it will show them
 
-```vue
-<NextQueries />
+```vue live
+<template>
```

Comment:
`<template>` can only have one root child. Wrap the components in a div for the code to work